targets/TARGET_STM/TARGET_STM32F0/TARGET_STM32F091xC/CMakeLists.txt

```diff
@@ -13,15 +11,19 @@ elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
     set(LINKER_FILE TOOLCHAIN_ARM/stm32f091xc.sct)
 endif()
 
-set_property(GLOBAL PROPERTY MBED_TARGET_LINKER_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
+add_library(STM32F091xC INTERFACE)
 
-target_sources(mbed-core
+target_sources(STM32F091xC
```

Comment:
these are mbed specific targets, why not  to prefix them with `mbed` as the rest of the components?